sequencer/src/bin/reset-storage.rs

```diff
@@ -21,6 +21,13 @@ struct Options {
 
 #[derive(Clone, Debug, Subcommand)]
 enum Command {
```

Comment:
Do we really need another binary for this? Can we just add the `reset` option to the binary that is using the db, as w/ solver?

Reply:
I think its easy for devops engineers to reset using this cli. I am not sure how we can add this reset functionality in sequencer and solver binary. cc @jbearer

Reply:
If we are not resetting the storage while running the sequencer then maybe we can remove this binary?

Reply:
I think sequencer could just have a --reset-db switch, and the only thing it would do is reset the db and exit.

Reply:
yeah we can do that. but lets keep it for now to unblock the deployment and we can remove it later if everyone agrees

Reply:
I'm concerned about the amount of binaries we currently have. I don't think we need this one. And I think deployment will get unblocked in the same amount of time whether this is a switch on sequencer or if it is a stand alone binary.

Reply:
I think it's a matter of personal preference. I like small and self-contained binaries that have a single responsibility. Of course a reset switch on the sequencer that resets and exits would also work, but if all it does is reset and exit, without running the actual sequencer logic, one might wonder why it is in the same binary as the sequencer.

Only really crucial thing here, though, is that if we make this a switch in the sequencer, it must exit after resetting. We used to have this as just a startup option on the sequencer, where it would reset the storage and then run the sequencer. This is bad because if the container then restarts for any reason, it will have the same config and thus will reset storage again unintentionally

Reply:
Agree that there is no hard and fast rule. I wouldn't be as concerned if there weren't already so many binaries.  This topic has been brought up in other conversations, so I think it is fair to begin to request that new binaries have sufficient justification.  I think it is fairly common-place to provide a way for a state-full application to clean up after itself. To me it seems strange to require someone to download a second binary to do this sort of house keeping. Also agree that it should exit after wipe.

Reply:
Maybe we can combine keygen, pub-key, and reset-storage into a single binary `utils`

Reply:
A binary is better for DevOps than a switch in the sequencer.  It's much easier to track and deploy, so we should keep the binaries, but it is fine to combine them into a single binary as Abdul said above.